scrapers_next/in/committees.py

```diff
@@ -0,0 +1,90 @@
+from spatula import URL, HtmlListPage, SkipItem, HtmlPage
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        if com:
+            for each_member in self.response.json()["members"]:
+                role = str(each_member["position"])
+                role = (
+                    role.lower()
+                    .replace("majority ", "")
+                    .replace("minority ", "")
+                    .title()
+                )
+                name = each_member["first_name"] + " " + each_member["last_name"]
```

Comment:
Let's be consistent with how we use strings. _Most_ strings in this scraper are f-strings...

Reply:
done...